### DISCUSSION TITLE
Would you like typo's in book reported ?

### DISCUSSION THREAD

**Discussion by Russsgithub** [savonet/liquidsoap#2056] [General]
If so where would you like them reported.

eg.

page 151

```
The http protocol actually denes two main ways of retrieving webpages: post, which
is handled by the function http.get presented above, and post, which is handled by the
function http.post. The post method is generally used for forms and takes an argument
named data, which contains the data we want to pass as the contents of the form. The way
this data is encoded is application-dependent and should be specied using the Content-Type
header. For instance, suppose that we have a script update_metadata.php that we can call to
update the metadata on our website.
```

should be

```
The http protocol actually denes two main ways of retrieving webpages: get, which
is handled by the function http.get presented above, and post, which is handled by the
function http.post. The post method is generally used for forms and takes an argument
named data, which contains the data we want to pass as the contents of the form. The way
this data is encoded is application-dependent and should be specied using the Content-Type
header. For instance, suppose that we have a script update_metadata.php that we can call to
update the metadata on our website.
```

**Comment by toots** [comment]
Thanks for these! I believe that @smimram is about to open-source the book, which should allow direct reports and/or PR there.

**Comment by Russsgithub** [comment]
OK.

On 14 Nov 2021 16:08, Romain Beauxis ***@***.***> wrote:

Thanks for these! I believe that @smimram<https://github.com/smimram> is about to open-source the book, which should allow direct reports and/or PR there.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/savonet/liquidsoap/discussions/2056#discussioncomment-1640427>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AB627QKFVBYO4WPJ7TPK5ITUL7NJNANCNFSM5H74J6MQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by smimram** [comment]
The sources are now available at https://github.com/savonet/book !

Please report there, possibly with a diff or PR :)

**Reply by Russsgithub** [reply]
Ok, will do.

**Reply by smimram** [reply]
Thanks!
